### PR TITLE
Added Profile page with current and past contests

### PR DESCRIPTION
**Pull request by eric-silva-61**
### What this PR does (required):
- Mock up for profile page with current and previous contests

### Screenshots / Videos (required):
profile1
profile2

### Any information needed to test this feature (required):
- This page is not accessible until the profile settings page is complete 

### Any issues with the current functionality (optional):
- None

### CODE REVIEW

File: client/src/components/ContestItem/useStyles.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  contestItemContainer: {
+    marginBottom: 25,
+  },
+  contestImg: {
+    //width: 150,
```
Comment (rajivtitus) [inline client/src/components/ContestItem/useStyles.ts:8]: Please remove

File: client/src/components/ContestItem/ContestItem.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <Grid item>
+        <ImageList cols={1}>
+          <ImageListItem key={1}>
+            <img src={imgSrc} alt={headline} className={classes.contestImg} />
+            <ImageListItemBar title={imgCount + ' SKETCH' + (imgCount > 1 ? 'ES' : '')} />
+          </ImageListItem>
+        </ImageList>
+      </Grid>
+      <Grid item className={classes.infoContainer}>
+        <Box className={classes.headline}>{headline}</Box>
```
Comment (rajivtitus) [inline client/src/components/ContestItem/ContestItem.tsx:34]: Maybe use a Typography here and the line below for increased semantics?

File: client/src/pages/ProfileSetting/Profile/Profile.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                imgSrc={contest.imgSrc}
+                imgCount={contest.imgCount}
+                headline={contest.headline}
+                byline={contest.byline}
+                prizeAmt={contest.prizeAmt}
+              />
+            ))}
+          </TabPanel>
+          <TabPanel value={value} index={1}>
+            {mockContests.map((contest, index) => (
```
Comment (rajivtitus) [inline client/src/pages/ProfileSetting/Profile/Profile.tsx:97]: Just an FYI for the future, when we have actual data please make sure to change indexes here :)

Ref: https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

**Review by rajivtitus — APPROVED** [review]
Great use of MUI, UI looks good too!

Just a couple of comments, please address before merging

File: client/src/components/ContestItem/useStyles.ts (hunk 1 above)
Comment (eric-silva-61) [reply client/src/components/ContestItem/useStyles.ts:8]: I've addressed this in #5

File: client/src/components/ContestItem/ContestItem.tsx (hunk 2 above)
Comment (eric-silva-61) [reply client/src/components/ContestItem/ContestItem.tsx:34]: I've addressed this in #5

File: client/src/pages/ProfileSetting/Profile/Profile.tsx (hunk 3 above)
Comment (eric-silva-61) [reply client/src/pages/ProfileSetting/Profile/Profile.tsx:97]: I've addressed this in #5

**Pull request merged**